### DISCUSSION TITLE
Generative Language Models for Danish

### DISCUSSION THREAD

**Discussion by KennethEnevoldsen** [centre-for-humanities-computing/danish-foundation-models#198] [Missing pieces for Danish NLP]
# Statement of need
Generative language models are useful and have multiple downstream use-cases. 

# Current status
The DFM project is currently working on developing dataset and training script for generative models for Danish

Related to #197

**Comment by renekim** [comment]
Hey hey, im NOT a resheacer ore anything, så jeg forsætter på dansk. Et lille spørgsmål fra en layman (også kaldet gøjler :D) Jeg kan sq ikke lige finde et sted hvor jeg kan downloade en mindre model som jeg kan køre lokalt (16 Gb - rtx4070) ?
Er det på tegnebrættet, eller er jeg WAAAY out of my League ?

mvh 
Renekim

**Reply by KennethEnevoldsen** [reply]
Hej Renekim, 

Der findes masser af modeller derude som du ville kunne køre på en 16 GB RTX. Googles gemma modeller er et eksempel.

Vi har mdodeller en finjusteres version af den ude fra som en del af Danish Dynaword (e.g. denne [her](https://huggingface.co/danish-foundation-models/gemma-3-1b-cpt-dynaword-full-v1))

Det er dog en research model så vi har ikke gjort meget for at gøre den produktionsvenlig, så hvis du ikke skal fine-tune den ville jeg nok bruge en anden model. [EuroEval](https://euroeval.com/leaderboards/Monolingual/danish/#__tabbed_1_3) er en god start.

Der er dog modeller på tegnebrættet. Den første release bliver dog nok en model trænet kun på "åbne data" (fully permissible licensed), hvilket vil påvirke performance men hvis man er varsom med hvilke modeller man sætter i produktion pga. træningsdata så er den safe. Vi forventer EU compliant modeller released efterfølgende.

Generelt kan du smide spørgsmål som denne ind på Danish Data Science Community også så kan du også høre hvad andre tænker :)

**Reply by renekim** [reply]
Tak for svaret. (Dette er et svar skrevet af ET MENNESKE !!!) :D
